datadog-crashtracker/build.rs

```diff
@@ -1,11 +1,45 @@
 // Copyright 2025-Present Datadog, Inc. https://www.datadoghq.com/
 // SPDX-License-Identifier: Apache-2.0
+#[cfg(unix)]
+use std::path::Path;
+
+pub use cc_utils::cc;
+
+#[cfg(unix)]
```

Comment:
If this doesn't work for macos, should it have a `#[cfg(not(osx))]` or whatever the format is?  

Do we document that this doen't work for osx